### PR TITLE
Added inline rugplot example #468

### PR DESCRIPTION
**Pull request by Ban-zee**
Added a rug plot example in the inline documentation.

### CODE REVIEW

File: arviz/plots/kdeplot.py [hunk 1]
```diff
@@ -80,16 +80,24 @@ def plot_kde(
...
 
     .. plot::
         :context: close-figs
 
         >>> import arviz as az
         >>> non_centered = az.load_arviz_data('non_centered_eight')
         >>> mu_posterior = np.concatenate(non_centered.posterior["mu"].values)
+        >>> az.plot_kde(mu_posterior, rug=True)
+
+    Default Kde without rugplot
```
Comment (ColCarroll) [inline arviz/plots/kdeplot.py]:
```suggestion
    Default KDE without rugplot
```

File: arviz/plots/kdeplot.py [hunk 2]
```diff
@@ -80,16 +80,24 @@ def plot_kde(
 
     Examples
     --------
-    Plot default KDE
+    Plot KDE with Rugplot
```
Comment (ColCarroll) [inline arviz/plots/kdeplot.py]:
```suggestion
    Plot KDE with rugplot
```

File: arviz/plots/kdeplot.py (hunk 1 above)
Comment (ColCarroll) [inline arviz/plots/kdeplot.py]: i would leave the "default plot" as the first example

File: arviz/plots/kdeplot.py [hunk 3]
```diff
@@ -80,16 +80,24 @@ def plot_kde(
...
 
     .. plot::
         :context: close-figs
 
         >>> import arviz as az
         >>> non_centered = az.load_arviz_data('non_centered_eight')
         >>> mu_posterior = np.concatenate(non_centered.posterior["mu"].values)
+        >>> az.plot_kde(mu_posterior, rug=True)
+
+    Default KDE without rugplot
```
Comment (ColCarroll) [inline arviz/plots/kdeplot.py]: this should still get moved to be the first example

File: arviz/plots/kdeplot.py (hunk 2 above)
Comment (ColCarroll) [reply arviz/plots/kdeplot.py]: Small change that will annoy me is that "rugplot" should not be capitalized

**Comment by ahartikainen** [comment]
Why was this PR closed?

**Comment by ahartikainen** [comment]
Hi, I don't think this PR should be closed, it is so close to finish.

If you need help making these changes we are more than happy to help.

**Comment by canyon289** [comment]
@Ban-zee Sorry to give you more trouble, but can I ask that you update your master branch with the repo master, than rebase your branch on top?

Currently this PR is bringing in too many changes not associated with rugplot. If my instructions are confusing let me know!

**Comment by Ban-zee** [comment]
No problem. I'll do so asap.

**Comment by canyon289** [comment]
There's no timelines this is open source :)

Yes, be sure to rebase your branch on top of the master branch from https://github.com/arviz-devs/arviz.

**Comment by canyon289** [comment]
This looks great! You'll need to fix one more error, a errant space at the end of one line but its almost there.

Thanks for sticking with this.

https://travis-ci.org/arviz-devs/arviz/jobs/484580642#L1902

File: arviz/plots/kdeplot.py [hunk 4]
```diff
@@ -80,7 +80,7 @@ def plot_kde(
 
     Examples
     --------
-    Plot default KDE
+    Plot default KDE
```
Comment (canyon289) [inline arviz/plots/kdeplot.py]: Lint is failing here due to trailing space

Comment (Ban-zee) [reply arviz/plots/kdeplot.py]: Thank you for pointing this out.

**Comment by canyon289** [comment]
LGTM. Thank you for the contribution!

**Comment by Ban-zee** [comment]
Thank you.

**Pull request merged**